java/net/anotheria/anosite/photoserver/api/photo/AlbumAO.java

```diff
@@ -47,6 +63,30 @@ public String getEncodedId() {
 		return IdCrypter.encode(getId());
 	}
 
+	public List<PhotoAO> getPhotos() {
+		return photos;
+	}
+
+	public void setPhotos(List<PhotoAO> photos) {
+		this.photos = photos;
+	}
+
+	public List<PhotoAO> getPhotosOrdered() {
+		Map<Long, PhotoAO> photosMap = new LinkedHashMap<>();
```

Comment:
strange stuff...
 in case if its executed more that 1 time during the AO - life time.. maybe better just create additional collection with this AO.... (simple copy of the incoming photos ( but not deep )) and  sort there ?